Compress.cpp

```diff
@@ -1001,6 +1012,9 @@ if (bin_filename[0] == '\0') {
 remove_extension(bin_filename, inf_filename);
 strcat(inf_filename, ".inf");
 
+remove_extension(bin_filename, inp_filename);
+strcat(inp_filename, ".inp");
```

Comment:
So my suggestion is to preserve `.inp` as the extension for packed inf files, but use `--pack-inf instead of `--pack-inp` for the command line.